Dockerfile

```diff
@@ -1,26 +1,16 @@
-FROM manageiq/ruby:latest
-
-RUN yum -y install centos-release-scl-rh && \
-    yum -y install --setopt=tsflags=nodocs \
-                   # To compile native gem extensions
-                   gcc-c++ \
-                   git \
-                   # To compile pg gem
-                   rh-postgresql10-postgresql-devel \
-                   rh-postgresql10-postgresql-libs \
-                   && \
-    yum clean all
+FROM registry.access.redhat.com/ubi8/ruby-25
+# This image runs as non-root by default.
+USER 0
```

Comment:
If it doesn't run as root, what is the "right" way to build using it? Surely every dockerfile can't be expected to include this line, right?

Or is this some kind of s2i thing?

Reply:
Yep it's all about the fact that this is also a s2i builder image. Switching this over to using the regular `ubi8/ubi` image fixed the problem.